.template-lintrc.js

```diff
@@ -9,14 +9,14 @@ module.exports = {
       The recommended rules are failing. Let's fix the errors
       and remove the rule exceptions one by one.
     */
-    'link-rel-noopener': false,
+    // 'link-rel-noopener': false,
```

Comment:
can you remove instead of commenting out?

Reply:
No problem! Just resubmitted.